### ISSUE TITLE
Error in tXml.d.ts 

### ISSUE THREAD

**Issue by grimmjo** [TobiasNickel/tXml#13]
Hi @TobiasNickel,

just got this issue while importing txml in my angular application.

```
Error: node_modules/txml/tXml.d.ts:25:23 - error TS1246: An interface property cannot have an initializer.

25     setPos: boolean = false;
```

Angular doesn't allow an initializer in interfaces :-/

Thanks for your help :-)

**Comment by TobiasNickel** [comment]
thanks, I will remove that, as it is false anyway. 

I just installed @angular/cli myself and create an app and installed txml. 

Did you also fix the ` Cannot find module 'stream'` error with `  "compilerOptions": { "types": ["node"] }` in `tsconfig.app.json`?

I will deploy this fix this weekend to npm.

**Comment by TobiasNickel** [comment]
ok, i published a new version 3.2.5 to npm. can you have a try?

**Comment by grimmjo** [comment]
Works like a charm ;) 

I use txml in a web worker. To fix the problem you have, just add `"types": ["node"]` to `tsconfig.app.json`. It's neither working in the `tsconfig.json` nor `tsconfig.base.json`.
